src/Network/Receive/kRO/Sakexe_0.pm

```diff
@@ -699,6 +767,11 @@ sub new {
 				types => 'a2 v C v V a8 l C',
 				keys => [qw(ID nameID type amount type_equip cards expire identified)],
 			},
+			type7 => { #PACKETVER_RE_NUM >= 20180704
```

Comment:
missing define type in
https://github.com/OpenKore/openkore/blob/aa42cd540c01b5b33c72dedf6b802efa384302a7/src/Network/Receive/ServerType0.pm#L766